pkg/controller/job/job_controller_test.go

```diff
@@ -725,14 +721,13 @@ func TestControllerSyncJob(t *testing.T) {
 	}
 
 	for name, tc := range testCases {
-		for _, wFinalizers := range []bool{false, true} {
-			t.Run(fmt.Sprintf("%s, finalizers=%t", name, wFinalizers), func(t *testing.T) {
+		// NOTE we should remove this but it makes the git diffs really ugly right now.
```

Comment:
It's not too bad.

There is a github configuration that makes this kind of changes easy to review
![image](https://user-images.githubusercontent.com/1299064/210603707-4889e252-e751-491b-a149-b67ccd973baf.png)